scripts/data/gen-nodetime/package.json

```diff
@@ -19,13 +16,20 @@
 		"js-yaml": "^4.0.0",
 		"json-rpc-2.0": "^0.2.16",
 		"long": "^4.0.0",
-		"pkg": "^4.4.9",
+		"pkg": "github:vercel/pkg#main",
```

Comment:
As soon as they release a new version we should pin it to that, but currently this is required.

Reply:
Is it possible if we can lock to a certain hash in `main` branch, so our builds can ensured to be deterministic?